test/integration/custom-routes/test/index.test.js

```diff
@@ -389,6 +417,22 @@ const runTests = (isDev = false) => {
             source: '/named-like-unnamed/:0',
             statusCode: 307,
           },
+          {
+            destination: '/another/:0/:test/:1',
+            regex: normalizeRegEx(
+              '^\\/unnamed-params\\/nested(?:\\/(.*?))(?:\\/([^\\/]+?))(?:\\/(.*))$'
+            ),
+            source: '/unnamed-params/nested/(.*?)/:test/(.*)',
+            statusCode: 307,
+          },
+          {
+            destination: '/:first*/:second*',
```

Comment:
This only needs to be the following right?
```suggestion
            destination: '/:first/:second',
```

Reply:
Right, updated